__tests__/app.test.js

```diff
@@ -129,3 +129,40 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("returns an array of comments for the given article_id", () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({body: {comments}}) => {
+            expect(comments.length).toBe(11)
+        })
+    })
+    test("comments should be formatted correctly", () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({body: {comments}}) => {
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                })
+            })
+        })
+    })
+    test("comments should be arranged most recent comments first", () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({body: {comments}}) => {
+            expect(comments).toBeSortedBy('created_at', { descending: false })
+        })
+    })
+    test("should return error 404 not found when given an invalid article ID", () => {
+        return request(app).get('/api/articles/99999/comments').expect(404).then(({body: {msg}}) => {
+            expect(msg).toEqual('Not Found')
+        })
+    })
+    test("should return error 400 Bad Request when given an invalid article ID data type", () => {
+        return request(app).get('/api/articles/forklift/comments').expect(400).then(({body: {msg}}) => {
+            expect(msg).toEqual('Bad Request')
+        })
+    })
```

Comment:
We should have a test for when the article id exists but there are no comments associated